bootcamp-java/Topic1/classes/Offers.java

```diff
@@ -0,0 +1,40 @@
+package classes;
+
+public class Offers {

```

Comment:
así es, aún no está terminada la programada pero sí modelada.La idea que tengo es que Offers extienda de Products y contenga una lista de productos.

Date: Tue, 20 Jan 2015 06:43:07 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] ShoppingCart (#4)

In bootcamp-java/Topic1/classes/Offers.java:

> @@ -0,0 +1,40 @@
> +package classes;
> +
> +public class Offers {

A qué aplicaría la oferta? Supongo que debería haber alguna relación con productos o algo así.


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =